forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -278,9 +292,13 @@ def post():
             mapper_schema = FormProcessMapperSchema()
             dict_data = mapper_schema.load(mapper_json)
             mapper = FormProcessMapperService.create_mapper(dict_data)
+
             FormProcessMapperService.unpublish_previous_mapper(dict_data)
+
             response = mapper_schema.dump(mapper)
             response["taskVariable"] = json.loads(response["taskVariable"])
+
+            FormHistoryService.created_form_logs_without_clone(data=mapper_json)
```

Comment:
As a best practice, the verbs in function names should be in the present tense (although there are some exceptions). suggestion `create_form_logs_without_clone`